src/main/webapp/scripts/perspective.js

```diff
@@ -0,0 +1,22 @@
+function getToxicity() {
+  API_KEY = '';
+  const commentText = document.getElementById("comment-text").value;
+  const commentObject = {
+    'comment': {'text': commentText},
+    'requestedAttributes': {'TOXICITY': {}},
+    'languages': ['en']
+  };
+  const json = JSON.stringify(commentObject);
+  const request = new Request(
+    `https://commentanalyzer.googleapis.com/v1alpha1/comments:analyze?key=${API_KEY}`,
+    {method: 'POST', body: json}
+  );
+  fetch(request)
+    .then(response => response.text())
+    .then((responseText) => {
+      const data = JSON.parse(responseText);
```

Comment:
not 100% sure but this should also work? 
```
fetch(request)
.then(response => response.json())
.then(data => {
const score = data.attributesScores ...
})
```

Reply:
I tried it earlier but I must've done something wrong because it works now haha, changed!